### ISSUE TITLE
Unable to log in after creating account

### ISSUE THREAD

**Issue by andrewhavens** [refinery/refinerycms-authentication-devise#5]
Just created my Refinery app a few days ago, just bundle updated refinery as well as this gem, and I keep getting redirected to the login screen when I try to log in using correct credentials. Reset password works though.

```
=> Booting WEBrick
=> Rails 4.2.3 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
[2015-07-19 14:22:45] INFO  WEBrick 1.3.1
[2015-07-19 14:22:45] INFO  ruby 2.2.2 (2015-04-13) [x86_64-darwin14]
[2015-07-19 14:22:45] INFO  WEBrick::HTTPServer#start: pid=69203 port=3000


Started GET "/admin/login" for ::1 at 2015-07-19 14:23:26 -0700
  ActiveRecord::SchemaMigration Load (0.1ms)  SELECT "schema_migrations".* FROM "schema_migrations"
Processing by Refinery::Authentication::Devise::SessionsController#new as HTML
  Parameters: {"locale"=>:en}
  Refinery::Authentication::Devise::Role Load (0.1ms)  SELECT  "refinery_authentication_devise_roles".* FROM "refinery_authentication_devise_roles" WHERE "refinery_authentication_devise_roles"."title" = ?  ORDER BY "refinery_authentication_devise_roles"."id" ASC LIMIT 1  [["title", "Refinery"]]
  Refinery::Authentication::Devise::User Exists (0.1ms)  SELECT  1 AS one FROM "refinery_authentication_devise_users" INNER JOIN "refinery_authentication_devise_roles_users" ON "refinery_authentication_devise_users"."id" = "refinery_authentication_devise_roles_users"."user_id" WHERE "refinery_authentication_devise_roles_users"."role_id" = ? LIMIT 1  [["role_id", 1]]
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_form_actions.html.erb (1.7ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/refinerycms-authentication-devise-1.0.1/app/views/refinery/authentication/devise/sessions/new.html.erb within refinery/layouts/login (31.3ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/_html_tag.html.erb (0.4ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_javascripts.html.erb (1064.8ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_head.html.erb (3467.5ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/_message.html.erb (0.4ms)
Completed 200 OK in 3602ms (Views: 3546.3ms | ActiveRecord: 0.9ms)
[2015-07-19 14:23:46] ERROR Errno::ECONNRESET: Connection reset by peer @ io_fillbuf - fd:17
	/Users/andrew/.rbenv/versions/2.2.2/lib/ruby/2.2.0/webrick/httpserver.rb:80:in `eof?'
	/Users/andrew/.rbenv/versions/2.2.2/lib/ruby/2.2.0/webrick/httpserver.rb:80:in `run'
	/Users/andrew/.rbenv/versions/2.2.2/lib/ruby/2.2.0/webrick/server.rb:294:in `block in start_thread'


Started POST "/admin/users/login" for ::1 at 2015-07-19 14:23:46 -0700
Processing by Refinery::Authentication::Devise::SessionsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"msDJQiajGsKaU++BGAW0a6lQsqp8NRwH3USrCMkROO8IuzYFfvUE+6wYLv02izd48G4zGo3O8/K3hSYO3Ok2sw==", "authentication_devise_user"=>{"login"=>"andrew", "password"=>"[FILTERED]", "remember_me"=>"0"}, "locale"=>:en}
  Refinery::Authentication::Devise::User Load (0.2ms)  SELECT  "refinery_authentication_devise_users".* FROM "refinery_authentication_devise_users" WHERE (username = 'andrew' OR email = 'andrew')  ORDER BY "refinery_authentication_devise_users"."id" ASC LIMIT 1
   (0.1ms)  begin transaction
  SQL (0.4ms)  UPDATE "refinery_authentication_devise_users" SET "last_sign_in_at" = ?, "current_sign_in_at" = ?, "sign_in_count" = ?, "updated_at" = ? WHERE "refinery_authentication_devise_users"."id" = ?  [["last_sign_in_at", "2015-07-19 20:39:45.709414"], ["current_sign_in_at", "2015-07-19 21:23:46.403579"], ["sign_in_count", 13], ["updated_at", "2015-07-19 21:23:46.404530"], ["id", 1]]
   (1.2ms)  commit transaction
Can't verify CSRF token authenticity
   (0.0ms)  begin transaction
   (0.0ms)  commit transaction
  Refinery::Authentication::Devise::Role Load (0.1ms)  SELECT  "refinery_authentication_devise_roles".* FROM "refinery_authentication_devise_roles" WHERE "refinery_authentication_devise_roles"."title" = ?  ORDER BY "refinery_authentication_devise_roles"."id" ASC LIMIT 1  [["title", "Refinery"]]
  Refinery::Authentication::Devise::User Exists (0.1ms)  SELECT  1 AS one FROM "refinery_authentication_devise_users" INNER JOIN "refinery_authentication_devise_roles_users" ON "refinery_authentication_devise_users"."id" = "refinery_authentication_devise_roles_users"."user_id" WHERE "refinery_authentication_devise_roles_users"."role_id" = ? LIMIT 1  [["role_id", 1]]
  Refinery::Authentication::Devise::User Load (0.4ms)  SELECT  "refinery_authentication_devise_users".* FROM "refinery_authentication_devise_users" WHERE (username = 'andrew' OR email = 'andrew')  ORDER BY "refinery_authentication_devise_users"."id" ASC LIMIT 1
   (0.1ms)  begin transaction
  SQL (0.3ms)  UPDATE "refinery_authentication_devise_users" SET "last_sign_in_at" = ?, "current_sign_in_at" = ?, "sign_in_count" = ?, "updated_at" = ? WHERE "refinery_authentication_devise_users"."id" = ?  [["last_sign_in_at", "2015-07-19 21:23:46.403579"], ["current_sign_in_at", "2015-07-19 21:23:46.482574"], ["sign_in_count", 14], ["updated_at", "2015-07-19 21:23:46.483223"], ["id", 1]]
   (0.6ms)  commit transaction
Devise scope: authentication_devise_user
Redirected to http://localhost:3000/admin
Completed 302 Found in 198ms (ActiveRecord: 3.4ms)


Started GET "/admin" for ::1 at 2015-07-19 14:23:46 -0700
Processing by Refinery::Admin::CoreController#index as HTML
  Parameters: {"locale"=>:en}
Redirected to http://localhost:3000/admin/login
Filter chain halted as :authenticate_refinery_user! rendered or redirected
Completed 302 Found in 3ms (ActiveRecord: 0.0ms)


Started GET "/admin/login" for ::1 at 2015-07-19 14:23:46 -0700
Processing by Refinery::Authentication::Devise::SessionsController#new as HTML
  Parameters: {"locale"=>:en}
  Refinery::Authentication::Devise::Role Load (0.1ms)  SELECT  "refinery_authentication_devise_roles".* FROM "refinery_authentication_devise_roles" WHERE "refinery_authentication_devise_roles"."title" = ?  ORDER BY "refinery_authentication_devise_roles"."id" ASC LIMIT 1  [["title", "Refinery"]]
  Refinery::Authentication::Devise::User Exists (0.1ms)  SELECT  1 AS one FROM "refinery_authentication_devise_users" INNER JOIN "refinery_authentication_devise_roles_users" ON "refinery_authentication_devise_users"."id" = "refinery_authentication_devise_roles_users"."user_id" WHERE "refinery_authentication_devise_roles_users"."role_id" = ? LIMIT 1  [["role_id", 1]]
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_form_actions.html.erb (0.2ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/refinerycms-authentication-devise-1.0.1/app/views/refinery/authentication/devise/sessions/new.html.erb within refinery/layouts/login (13.8ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/_html_tag.html.erb (0.1ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_javascripts.html.erb (203.6ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/admin/_head.html.erb (259.7ms)
  Rendered /Users/andrew/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/refinerycms-9253deaefe73/core/app/views/refinery/_message.html.erb (0.1ms)
Completed 200 OK in 316ms (Views: 312.8ms | ActiveRecord: 0.2ms)
```

**Comment by bricesanchez** [comment]
Could you paste your Gemfile? We've made a lot of change a few days ago.

**Comment by simi** [comment]
Gemfile.lock will be better here IMHO.

2015-07-19 23:35 GMT+02:00 Brice Sanchez <notifications@github.com>:

**Comment by bricesanchez** [comment]
Yes @simi

**Comment by andrewhavens** [comment]
```
GIT
  remote: https://github.com/refinery/refinerycms
  revision: 9253deaefe7351913cfbf149033773254361f0f0
  branch: master
  specs:
    refinerycms (3.0.0)
      refinerycms-core (= 3.0.0)
      refinerycms-images (= 3.0.0)
      refinerycms-pages (= 3.0.0)
      refinerycms-resources (= 3.0.0)
    refinerycms-core (3.0.0)
      actionpack (>= 4.1.5, < 5.0)
      activerecord (>= 4.1.5, < 5.0)
      awesome_nested_set (~> 3.0.0)
      coffee-rails (~> 4.0, >= 4.0.0)
      decorators (~> 2.0.0)
      jquery-rails (>= 2.3.0)
      jquery-ui-rails (~> 5.0.0)
      railties (>= 4.1.5, < 5.0)
      refinerycms-i18n (~> 3.0.0)
      sass-rails (>= 4.0, < 5.1)
      truncate_html (~> 0.9)
      will_paginate (~> 3.0.2)
      zilch-authorisation
    refinerycms-images (3.0.0)
      dragonfly (~> 1.0.0)
      refinerycms-core (= 3.0.0)
    refinerycms-pages (3.0.0)
      awesome_nested_set (~> 3.0.0)
      babosa (!= 0.3.6)
      friendly_id (~> 5.1.0)
      globalize (>= 4.0.0, < 5.2)
      refinerycms-core (= 3.0.0)
      seo_meta (~> 2.0.0.rc.1)
      speakingurl-rails (~> 1.1.2)
    refinerycms-resources (3.0.0)
      acts_as_indexed (~> 0.8.0)
      dragonfly (~> 1.0.0)
      refinerycms-core (= 3.0.0)

GEM
  remote: https://rubygems.org/
  specs:
    actionmailer (4.2.3)
      actionpack (= 4.2.3)
      actionview (= 4.2.3)
      activejob (= 4.2.3)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.3)
      actionview (= 4.2.3)
      activesupport (= 4.2.3)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.3)
      activesupport (= 4.2.3)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (4.2.3)
      activesupport (= 4.2.3)
      globalid (>= 0.3.0)
    activemodel (4.2.3)
      activesupport (= 4.2.3)
      builder (~> 3.1)
    activerecord (4.2.3)
      activemodel (= 4.2.3)
      activesupport (= 4.2.3)
      arel (~> 6.0)
    activesupport (4.2.3)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    acts_as_indexed (0.8.3)
    addressable (2.3.8)
    arel (6.0.2)
    awesome_nested_set (3.0.2)
      activerecord (>= 4.0.0, < 5)
    babosa (1.0.2)
    bcrypt (3.1.10)
    binding_of_caller (0.7.2)
      debug_inspector (>= 0.0.1)
    builder (3.2.2)
    byebug (5.0.0)
      columnize (= 0.9.0)
    coffee-rails (4.1.0)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.0)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.9.1.1)
    columnize (0.9.0)
    debug_inspector (0.0.2)
    decorators (2.0.1)
      railties (>= 4.0.0, < 5.0.0)
    devise (3.5.1)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 3.2.6, < 5)
      responders
      thread_safe (~> 0.1)
      warden (~> 1.2.3)
    dragonfly (1.0.10)
      addressable (~> 2.3)
      multi_json (~> 1.0)
      rack (>= 1.3.0)
    erubis (2.7.0)
    execjs (2.5.2)
    friendly_id (5.1.0)
      activerecord (>= 4.0.0)
    globalid (0.3.5)
      activesupport (>= 4.1.0)
    globalize (5.0.1)
      activemodel (>= 4.2.0, < 4.3)
      activerecord (>= 4.2.0, < 4.3)
    i18n (0.7.0)
    jbuilder (2.3.1)
      activesupport (>= 3.0.0, < 5)
      multi_json (~> 1.2)
    jquery-rails (4.0.4)
      rails-dom-testing (~> 1.0)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    jquery-ui-rails (5.0.5)
      railties (>= 3.2.16)
    json (1.8.3)
    loofah (2.0.2)
      nokogiri (>= 1.5.9)
    mail (2.6.3)
      mime-types (>= 1.16, < 3)
    mime-types (2.6.1)
    mini_portile (0.6.2)
    minitest (5.7.0)
    multi_json (1.11.2)
    nokogiri (1.6.6.2)
      mini_portile (~> 0.6.0)
    orm_adapter (0.5.0)
    quiet_assets (1.1.0)
      railties (>= 3.1, < 5.0)
    rack (1.6.4)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.3)
      actionmailer (= 4.2.3)
      actionpack (= 4.2.3)
      actionview (= 4.2.3)
      activejob (= 4.2.3)
      activemodel (= 4.2.3)
      activerecord (= 4.2.3)
      activesupport (= 4.2.3)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.3)
      sprockets-rails
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.6)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-html-sanitizer (1.0.2)
      loofah (~> 2.0)
    rails-i18n (4.0.4)
      i18n (~> 0.6)
      railties (~> 4.0)
    railties (4.2.3)
      actionpack (= 4.2.3)
      activesupport (= 4.2.3)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (10.4.2)
    rdoc (4.2.0)
    refinerycms-acts-as-indexed (2.0.1)
      acts_as_indexed (~> 0.8.0)
      decorators (~> 2.0.0)
      refinerycms-core (~> 3.0.0)
    refinerycms-authentication-devise (1.0.1)
      actionmailer (~> 4.2, >= 4.2.0)
      devise (~> 3.0, >= 3.2.4)
      friendly_id (~> 5.1.0)
      refinerycms-core (~> 3.0, >= 3.0.0)
    refinerycms-i18n (3.0.1)
      rails-i18n (~> 4.0.1)
      routing-filter (>= 0.4.0)
    refinerycms-wymeditor (1.0.6)
      refinerycms-core (~> 3.0, >= 3.0.0)
    responders (2.1.0)
      railties (>= 4.2.0, < 5)
    routing-filter (0.5.0)
      actionpack (~> 4.2)
      activesupport (~> 4.2)
    sass (3.4.16)
    sass-rails (5.0.3)
      railties (>= 4.0.0, < 5.0)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (~> 1.1)
    sdoc (0.4.1)
      json (~> 1.7, >= 1.7.7)
      rdoc (~> 4.0)
    seo_meta (2.0.0.rc.1)
      railties (>= 3.0.0)
    speakingurl-rails (1.1.5)
      railties (>= 3.1)
    spring (1.3.6)
    sprockets (3.2.0)
      rack (~> 1.0)
    sprockets-rails (2.3.2)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      sprockets (>= 2.8, < 4.0)
    sqlite3 (1.3.10)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (1.4.1)
    truncate_html (0.9.3)
    turbolinks (2.5.3)
      coffee-rails
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (2.7.1)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    warden (1.2.3)
      rack (>= 1.0)
    web-console (2.2.1)
      activemodel (>= 4.0)
      binding_of_caller (>= 0.7.2)
      railties (>= 4.0)
      sprockets-rails (>= 2.0, < 4.0)
    will_paginate (3.0.7)
    zilch-authorisation (0.0.1)

PLATFORMS
  ruby

DEPENDENCIES
  byebug
  coffee-rails (~> 4.1.0)
  jbuilder (~> 2.0)
  jquery-rails
  quiet_assets
  rails (= 4.2.3)
  refinerycms!
  refinerycms-acts-as-indexed (~> 2.0, >= 2.0.0)
  refinerycms-authentication-devise (~> 1.0)
  refinerycms-wymeditor (~> 1.0, >= 1.0.6)
  sass-rails (~> 5.0)
  sdoc (~> 0.4.0)
  spring
  sqlite3
  turbolinks
  uglifier (>= 1.3.0)
  web-console (~> 2.0)

BUNDLED WITH
   1.10.5
```

**Comment by andrewhavens** [comment]
```
source 'https://rubygems.org'


# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.3'
# Use sqlite3 as the database for Active Record
group :development, :test do
  gem 'sqlite3'
end
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use jquery as the JavaScript library
gem 'jquery-rails'
# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
gem 'turbolinks'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.0'
# bundle exec rake doc:rails generates the API under doc/api.
gem 'sdoc', '~> 0.4.0', group: :doc

# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'

  # Access an IRB console on exception pages or by using <%= console %> in views
  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end


gem 'refinerycms', git: 'https://github.com/refinery/refinerycms', branch: 'master'

gem 'quiet_assets', group: :development

# Add support for searching inside Refinery's admin interface.
gem 'refinerycms-acts-as-indexed', ['~> 2.0', '>= 2.0.0']

# Add support for Refinery's custom fork of the visual editor WYMeditor.
gem 'refinerycms-wymeditor', ['~> 1.0', '>= 1.0.6']

# The default authentication adapter
gem 'refinerycms-authentication-devise', '~> 1.0'
```

**Comment by bricesanchez** [comment]
I didn't have this error with https://github.com/bricesanchez/spree-refinery-authentication. 

I could try tomorrow after a bundle update.

**Comment by bricesanchez** [comment]
It works for me on my project with spree and spree-refinery-authentication.

**Comment by nzgrover** [comment]
Critical line is "Can't verify CSRF token authenticity". Works when you are creating a new user, but not for an subsequent log in.

**Comment by nntr** [comment]
Hi,
I faced with the same bug. 
Looks that I found the root cause. 
Devise will delete _csrf_token in subsequent calls of detect_authentication_devise_user and this will lead to "Can't verify CSRF..." during login. See callchain:

> after delete _csrf_token {"session_id"=>"eeb996e15bfac4fbac8857e215186e14", "flash"=>{"discard"=>["notice"], "flashes"=> {"notice"=>"Signed out successfully."}}, "return_to"=>"/refinery", "warden.user.authentication_devise_user.key"=>[[1], "$2a$10$Xp9PMlQWx8D.osz4y22Zbe"]}
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/activesupport-4.2.3/lib/active_support/core_ext/object/try.rb:77:in `public_send'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/activesupport-4.2.3/lib/active_support/core_ext/object/try.rb:77:in `try!'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/activesupport-4.2.3/lib/active_support/core_ext/object/try.rb:63:in `try'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/devise-3.5.1/lib/devise/hooks/csrf_cleaner.rb:5:in `block in <top (required)>'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/hooks.rb:14:in `call'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/hooks.rb:14:in `block in _run_callbacks'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/hooks.rb:9:in `each'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/hooks.rb:9:in `_run_callbacks'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/manager.rb:53:in `_run_callbacks'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/proxy.rb:179:in `set_user'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/proxy.rb:323:in `_perform_authentication'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/warden-1.2.3/lib/warden/proxy.rb:104:in `authenticate'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/devise-3.5.1/lib/devise/controllers/helpers.rb:120:in `current_authentication_devise_user'
> C:/Ruby21/lib/ruby/gems/2.1.0/gems/refinerycms-authentication-devise-1.0.1/app/decorators/controllers/action_controller_base_decorator.rb:19:in `detect_authentication_devise_user!'

**Comment by nntr** [comment]
Please see my pull request. I'm new at ruby/rails and probably this is not the best solution.

**Comment by parndt** [comment]
Fixed by #7 - wonderful solution!

**Comment by parndt** [comment]
Please update to version 1.0.2 or above (if you're reading this later when/if there are more versions).
